src/query/sql/src/planner/binder/copy_into_table.rs

```diff
@@ -561,7 +561,7 @@ pub async fn resolve_file_location(
     match location.clone() {
         FileLocation::Stage(location) => resolve_stage_location(ctx, &location).await,
         FileLocation::Uri(mut uri) => {
-            let (storage_params, path) = parse_uri_location(&mut uri).await?;
+            let (storage_params, path) = parse_uri_location(&mut uri, Some(ctx)).await?;
```

Comment:
Is it a good idea to always passing a ctx to simplify the usage?

Reply:
@Xuanwo 2 cases need None:
1. create connection
2. tests (not necessary, but otherwise need a fake table context.)